src/readme.html

```diff
@@ -3,96 +3,96 @@
-		<dd>If you&#8217;ve looked everywhere and still can&#8217;t find an answer, the support forums are very active and have a large community ready to help. To help them help you be sure to use a descriptive thread title and describe your question in as much detail as possible.</dd>
-	<dt><a href="https://codex.wordpress.org/IRC">WordPress <abbr title="Internet Relay Chat">IRC</abbr> Channel</a></dt>
-		<dd>There is an online chat channel that is used for discussion among people who use WordPress and occasionally support topics. The above wiki page should point you in the right direction. (<a href="irc://irc.freenode.net/wordpress">irc.freenode.net #wordpress</a>)</dd>
+	<dt><a href="https://docs.classicpress.net/">ClassicPress Documentation</a></dt>
+		<dd>Slowly but steadily our online documentation is growing.</dd>
+	<dt><a href="https://www.classicpress.net/blog/">ClassicPress Blog</a></dt>
+		<dd>This is where you&#8217;ll find the latest updates and news related to ClassicPress. Recent ClassicPress news appears in your administrative dashboard by default.</dd>
+	<dt><a href="https://forums.classicpress.net/c/support">ClassicPress Support Forums</a></dt>
+		<dd>The support forums are a great way to get help from other users and of course the different ClassicPress moderators are also active to lend a helping hand.</dd>
+	<dt><a href="https://www.classicpress.net/join-slack/">ClassicPress Slack</a></dt>
+		<dd>There is an online chat channel that is used for discussion among people who use ClassicPress and occasionally support topics.</dd>
+	<dt><a href="https://petitions.classicpress.net/">ClassicPress Petitions</a></dt>
+		<dd>Please share and vote for ideas to be included in ClassicPress 2.0.0. Suggestions that reach the threshold will be put to an official vote by the wider community.</dd>
+	<dt><a href="https://www.classicpress.net/democracy/">ClassicPress Democracy</a></dt>
+		<dd>Last but not least it is good to read about the ClassicPress Democracy. ClassicPress is a democratic community-led fork of WordPress that enables all stakeholders to shape the direction that the project takes. This page aims to explain how we handle this process to ensure that power doesn’t become centralised and that every voice can be heard.</dd>
 </dl>
 
 <h2>Final Notes</h2>
-<ul>
-	<li>If you have any suggestions, ideas, or comments, or if you (gasp!) found a bug, join us in the <a href="https://forums.classicpress.net/c/support">Support Forums</a>.</li>
-	<li>WordPress has a robust plugin <abbr title="application programming interface">API</abbr> that makes extending the code easy. If you are a developer interested in utilizing this, see the <a href="https://developer.wordpress.org/plugins/">Plugin Developer Handbook</a>. You shouldn&#8217;t modify any of the core code.</li>
+<p>If you have any suggestions, ideas, or comments, or if you found a bug, join us in the <a href="https://forums.classicpress.net/c/support">Support Forums</a>.</p>
 </ul>
```

Comment:
The opening tag for this `ul` was removed, so the closing `</ul>` needs to be removed too.

Reply:
sorry about that